openpype/hosts/maya/plugins/publish/collect_render.py

```diff
@@ -326,8 +326,8 @@ def process(self, context):
                 "byFrameStep": int(
                     self.get_render_attribute("byFrameStep",
                                               layer=layer_name)),
-                "renderer": self.get_render_attribute("currentRenderer",
-                                                      layer=layer_name),
+                "renderer": self.get_render_attribute(
+                    "currentRenderer", layer=layer_name).lower(),
```

Comment:
I'm a bit worried about forcing this to lowercase here - what if the render submission to a render manager requires the original Renderer name (including uppercase characters?) for the submission.

Deadline's MayaBatch plug-in seems to check against lowercase (except for a single `self.Renderer == "redshiftExport"` check which I think  is just a bug in Deadline's code since other mentions do use lowercase for it too.)

Would we possibly run ourselves into issues due to us forcing lowercase?

Reply:
yeah, I was looking into it too. I think we are using renderer name in lower case everywhere and this particular case was the one and only that wasn't using lower case name of renderer. I vote for addressing this in another issue if necessary - this works on Deadline but we might hit problems when using other farms like RoyalRender or Muster where the renderer name can be completely different or case sensitive and it that case it is easier to create mapping between our lower case renderer and farm specific names.